obfuscator-core/src/main/java/me/superblaubeere27/jobf/ui/GUI.java

```diff
@@ -79,7 +79,7 @@ public GUI(String updateCheckResult) {
         setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
         setContentPane(panel1);
         Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
-        setSize((int) (screenSize.getWidth() / 2), (int) (screenSize.getHeight() / 1.9));
+        setSize((int) (screenSize.getWidth() / 2), (int) (screenSize.getHeight() / 1.7));
```

Comment:
Had to change this, because the flatlaf tabs height it taller than, default one.